src/main/fc/fc_msp.c

```diff
@@ -287,7 +288,12 @@ static void serializeDataflashSummaryReply(sbuf_t *dst)
 }
 
 #ifdef USE_FLASHFS
-static void serializeDataflashReadReply(sbuf_t *dst, uint32_t address, const uint16_t size, bool useLegacyFormat)
+enum compressionType_e {
+    NO_COMPRESSION,
+    HUFFMAN
+};
+
+static void __attribute__((optimize("-O0"))) serializeDataflashReadReply(sbuf_t *dst, uint32_t address, const uint16_t size, bool useLegacyFormat)
```

Comment:
Yes, tried to get local variables to display properly in GDB, but current master shows some rubbish even with optimizations off. It wasn't like that 2 weeks ago :_(